lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -354,6 +354,21 @@
     CODEGEN_ROOT: "{{ codegen_root }}"
   when: preburn
 
+- name: Download gHZ binary from artifactory
+  get_url:
+    url: https://artifactory.magmacore.org:443/artifactory/blob-prod/ghz
+    dest: "/tmp"
```

Comment:
you can directly get the binary at the right sport using `dest: /usr/local/bin/ghz` and mode to set up the correct rights